tests/test_utils.py

```diff
@@ -0,0 +1,122 @@
+import json
+import os
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+
+from averell.utils import download_corpora
+from averell.utils import download_corpus
+from averell.utils import filter_features
+from averell.utils import get_line_features
+from averell.utils import get_stanza_features
+from averell.utils import get_syllable_features
+from averell.utils import get_word_features
+
+TESTS_DIR = Path(os.path.dirname(os.path.realpath(__file__)))
+
+
+@pytest.fixture
+def zip_test():
+    return str(Path.cwd() / "tests" / "fixtures" / "test.zip")
+
+
+@pytest.fixture
+def stanza_features():
+    return json.loads(Path("tests/fixtures/stanza_features.json").read_text())
```

Comment:
Try to avoid tests/fixtures/disco.json and use a proper Path() instead for win/unix compatibility

Reply:
Done